Frameworks/UIKit/UIImage.mm

```diff
@@ -123,6 +122,11 @@ + (UIImageCachedObject*)cacheImage:(UIImage*)image withName:(NSString*)name {
     return [obj autorelease];
 }
 
+static inline CGImageRef getImage(UIImage* self) {
+    RETURN_NULL_IF(!self);
```

Comment:
Given some of the conditions (e.g) initWithImage, can return nil.  Given getImage is used in multiple places, the return nil is useful.

---
In reply to: [84748990](https://github.com/Microsoft/WinObjC/pull/1185#discussion_r84748990) [](ancestors = 84748990)